rm/rm-server/build.gradle

```diff
@@ -16,6 +16,7 @@ dependencies {
     )
 
     compile 'com.google.guava:guava:19.0'
+    compile group: 'org.javassist', name: 'javassist', version: '3.15.0-GA'
```

Comment:
programming dependencies should already include javassist, normally we should reuse this version and not overwrite it

Reply:
Done. It's removed.